### DISCUSSION TITLE
Setting up wind for a selected region

### DISCUSSION THREAD

**Discussion by R-Godakandage** [firemodels/fds#13084] [Q&A] [answered]
In FDS is there any option that we can set up wind for a part of the domain? Because I see I at the beginning we have wind within modelled room compartments as well. Your feedback/ advice is highly appreciated

**Comment by rmcdermo** [comment, marked as answer]
Yes, have a look at the user guide section 20.6 Nonuniform Initial Fields: Velocity, Temperature, Species.   Also see the CSVF table at the back of the guide.  It is not very easy, but it is the only way we've come up with the make this general.

**Reply by R-Godakandage** [reply]
Thanks a lot Dr. McDermott for the information.
